agent/src/agentclient.rs

```diff
@@ -137,15 +191,15 @@ impl<T: APIServerClient> BrupopAgent<T> {
 
     /// update the custom resource associated with this node
     async fn update_metadata_custom_resource(
-        &mut self,
-        current_metadata: BottlerocketNodeStatus,
+        &self,
+        _current_metadata: BottlerocketNodeStatus,
```

Comment:
Curious why do you add underscore here? I thought only variable that you won't use start with underscore.

Reply:
Yeah. I think I should remove it. Somehow `Rust-analyzer` suggests me to add `_`. thanks